src/utils/parser.py

```diff
@@ -1,21 +1,16 @@
 import re
+import difflib
 
-COMMANDS_RGX = 'hello|close|exit|good bye|' \
-               'add contact|' \
-               'change phone|' \
-               'find contact by name|' \
-               'show all|' \
-               'add extra phone|' \
-               'find phone|' \
-               'delete contact|' \
-               'remove phone|' \
-               'find birthday|' \
-               'find by key'
+COMMANDS = ['hi', 'hello', 'close', 'exit', 'good bye', 'add contact', 'find birthday', 'find contact', 'remove phone',
```

Comment:
I would suggest moving COMMANDS_RGX from the parser and BOT_COMMANDS from main to globals and importing them from there. this will make the code cleaner and more coherent and allow for easier changes